src/server/common/helpers/token/token-utils.js

```diff
@@ -0,0 +1,22 @@
+import { token } from '@hapi/jwt'
+
+const SECRET_LENGTH = 36
+const ISSUER_START_OFFSET = 73
+const ISSUER_END_OFFSET = 37
+
+export const secrets = (apiKey) => ({
+  secret: apiKey.substring(apiKey.length - SECRET_LENGTH, apiKey.length),
+  iss: apiKey.substring(
+    apiKey.length - ISSUER_START_OFFSET,
+    apiKey.length - ISSUER_END_OFFSET
+  )
+})
+
+export function createToken(apiKey) {
+  const iat = Math.round(Date.now() / 1000)
+  const { iss, secret } = secrets(apiKey)
+
+  return token.generate({ iss, iat }, secret, {
+    header: { typ: 'JWT', alg: 'HS256' }
+  })
+}
```

Comment:
This token util is (largely) specific to the way that notify handles its tokens, so should be in the same folder with the connector IMO.  Namespace it with notify.